scipy/linalg/flapack_gen_tri.pyf.src

```diff
@@ -66,6 +66,32 @@ subroutine <prefix>gttrs(trans, n, nrhs, dl, d, du, du2, ipiv, b, ldb, info)
 end subroutine gttrs
 
 
+subroutine <prefix>gtcon(norm,n,dl,d,du,du2,ipiv,anorm,rcond,work,iwork,info)
```

Comment:
In [`?gecon`](https://github.com/scipy/scipy/blob/87c46641a8b3b5b47b81de44c07b840468f7ebe7/scipy/linalg/flapack_gen.pyf.src#L168), the second to last argument is also called `iwork`, but in the [wrapper](https://github.com/scipy/scipy/blob/87c46641a8b3b5b47b81de44c07b840468f7ebe7/scipy/linalg/flapack_gen.pyf.src#L168), it is called `irwork`. I've seen this several times. Why is it renamed?